### PR TITLE
skip gcp tests

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [ ] documentation
- [ ] dependencies

### CODE REVIEW

File: tests/integration/cloud_governance/common/clouds/gcp/test_google_account.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
 from datetime import datetime, timedelta
 
+import pytest
+
 from cloud_governance.common.clouds.gcp.google_account import GoogleAccount
 from cloud_governance.main.environment_variables import environment_variables
 
 
+@pytest.mark.skip(reason="Data is not being exported to the database")
```
Comment (ebattat) [inline tests/integration/cloud_governance/common/clouds/gcp/test_google_account.py:9]: Can you add comment with Ticket id ?

**Comment by athiruma** [comment]
Case 55659104: The database table is not updated

File: tests/integration/cloud_governance/common/clouds/gcp/test_google_account.py (hunk 1 above)
Comment (ebattat) [reply tests/integration/cloud_governance/common/clouds/gcp/test_google_account.py:9]: @athiruma, I dont see the comment in the code, where u add it ?

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**